code-of-conduct.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
+	<!-- ## SITE META ## -->
+	<meta charset="utf-8">
+	<title>Code of Conduct | 2017 Write/Speak/Code Conference</title>
+	<meta name="description" content="Our event and its associated online spaces are dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.">
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+	<link rel="shortcut icon" type="image/x-icon" href="assets/img/w-s-c-favicon.ico">
+	<!-- ## LOAD STYLSHEETS ## -->
+	<link rel="stylesheet" media="all" href="./assets/css/style.css"/>
+	<link rel="stylesheet" media="all" href="./assets/css/custom.css"/>
+	<!-- ## FLEXBOX GRID ## -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css" type="text/css" >
+	<!-- ## GOOGLE FONTS ## -->
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans|PT+Sans" rel="stylesheet">
+</head>
+
+<body class="code-of-conduct-page">
+  <section class="page-intro">
+    <header class="row center-xs">
+      <div class="col-xs-12">
+        <a href="http://www.writespeakcode.com" target="_blank">
+          <img src="./assets/img/WSClogos_stacked_coral-blush-gold_screen.png" width="200" alt="Write Speak Code logo" class="wsc-logo"/>
+        </a>
+        <h1>Conference Code of Conduct</h1>
+        <a href="index.html" target="_blank" class="button secondary">back to conference website</a>
+      </div>
```

Comment:
i like the CoC opening in a new tab but having the conf website open in a new tab from here = lots of tabs. what do you think?